tests/parse/test_parse_variant.py

```diff
@@ -103,3 +103,14 @@ def test_parse_mitomapassociateddiseases(variants, case_obj):
             parsed_variant = parse_variant(variant, case_obj)
             # THEN make sure that it is parsed correct
             assert parsed_variant["mitomap_associated_diseases"] == mitomap_associated_diseases
+
+
+def test_parse_hmtvar(variants, case_obj):
+    # GIVEN some parsed variant dicts
+    for variant in variants:
+        # WHEN "mitomap associated diseases" is present
+        if variant.INFO.get("HmtVar"):
```

Comment:
Tests are much appreciated! 🥇  

For how it's written I realize that if one day we modify scout/demo/643594.clinical.vcf.gz by removing the MT variant containing HmtVar and MITOMAP info, the relative tests will still be successful because you'll never enter the condition in this line, and line 101 in the MITOMAP test.

Perhaps a better way of testing the parsing would be something like this ?
```
def test_parse_hmtvar(cyvcf2_variant, case_obj):
    """Test parsing HmtVar value from variant annotated with HmtNote"""

    # GIVEN a variant containing HmtVar key in the INFO field:
    cyvcf2_variant.INFO["HmtVar"] = "39192"

    # THEN make sure that it is parsed correctly
    hmtvar_variant_id = int(cyvcf2_variant.INFO["HmtVar"])
    parsed_variant = parse_variant(cyvcf2_variant, case_obj)    
    assert parsed_variant["hmtvar_variant_id"] == hmtvar_variant_id
```

Reply:
I agree that silently "failing" tests should be avoided, but I also think it feels more realistic to parse an actual file. But I don't have a better suggestion, so I'll change it according to your suggestion (tomorrow)! :)

Reply:
An alternative would be do the parsing of the real file and then run a mongo query to look for a variant with those specifics..

Reply:
I took the easy way out and implemented them according to your first suggestion :)